doc/content/getting-started/user-management/_index.md

```diff
@@ -4,16 +4,26 @@ description: ""
 aliases: [/user-management]
 ---
 
-{{% tts %}} allows network administrators to manage the users that have access to the network. Administrators may add and remove users, and perform any actions on Gateways, End Devices, and Applications.
-
-For non-administrator users, {{% tts %}} manages access rights to Gateways, End Devices, and Applications through collaborations.
-
 This section explains how to manage users and organizations, and how to grant users and organizations rights as collaborators of entities.
 
 <!--more-->
 
+## Administrator Rights
+
+{{% tts %}} allows network administrators to manage the users that have access to the network. Administrators may:
+
+- Manage users in the tenant
```

Comment:
"in the tenant" only applies to multi-tenant. For single-tenant it's "in the network"